style-guide.md

```diff
@@ -0,0 +1,109 @@
+# Style Guide #
+
+Here lie the heretofore-unwritten rules and styles we use when writing
+documentation, issues, pull requests, and other forms of written expression.
+
+## References ##
+
+- [Plain Language guide](https://www.plainlanguage.gov/guidelines/)
+- [GitHub list of supported languages](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml)
+
+## Repositories ##
+
+**All repositories should be public.**
+
+Whether creating new repositories or transferring existing repositories,
+remember that we work in the open by default. For more, when we might use
+private repositories, read our [open source policy](https://github.com/cisagov/development-guide/tree/develop/open-source-policy).
+
+### Repository naming ###
+
+Use a repository's name to describe its purpose. Names should be:
+
+1. Descriptive
+1. Readable
+1. Consistent
+1. Contextual
+1. Future-friendly
+1. Extensible
+1. Reusable
+
+Avoid references to pop culture or other implicit references. GitLab has a
+good guide on using [inclusive language in documentation](https://docs.gitlab.com/ee/development/documentation/styleguide/#inclusive-language).
+
+Specifically, when naming a repository:
+
+- Use lower case
+- Use dashes
+  - For readability, where spaces would otherwise separate words
+  - Avoid underscores, which require extra keystrokes
+
+### Repository settings ###
+
+Specify code owners in each repository's `.github/codeowners` file to make
+sure that changes and pull requests are reviewed by people familiar with
+the codebase. This way, we'll catch more problems before they get into the
+protected branch.
+
+- Set a minimum of two code owners for all repositories.
+  - Add or edit each repository's `.github/codeowners` file.
+- Set up [branch protection](project_setup/branch-protection.md).
+
+## Headers and headings ##
+
+Use sentence case. For example:
+
+- `# Use variables to configure pipelines`
+- `## Use the Style Guide`
```

Comment:
Historically we have pushed for opening _and closing_ octothorpes.  This is probably worth mentioning in the text for this section, as well as making this change in your examples.
```suggestion
- `# Use variables to configure pipelines #`
- `## Use the Style Guide ##`
```

Reply:
It appears that sentence is not actually used for "first-level" (one octothorpe) headings.  This should be documented.